tcms/testruns/tests/test_admin.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+from django.urls import reverse
+from tcms.tests import LoggedInTestCase
+from tcms.utils.permissions import initiate_user_with_default_setups
+
+
+class TestTestRunAdmin(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        initiate_user_with_default_setups(cls.tester)
+
+    def test_regular_user_can_not_add_view(self):
```

Comment:
"Can not add" what ? The last `_view` is not needed nor does it make any sense. Either "can not add testrun" or just "can not add" b/c we already know where we are and which page the test class is trying to verify.